gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -182,7 +158,7 @@ public String selector() {
   /**
    * Class for specifying record set listing options.
    */
-  class RecordSetListOption extends AbstractOption implements Serializable {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://www.codacy.com/app/mziccard/gcloud-java/file/2326222365/issues/source?bid=3123103&fileBranchId=3204660#l161)